_posts/2021-12-31-2021-in-numbers.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Maxwell Engineering in 2021 in numbers
+author_avatar: https://avatars.githubusercontent.com/u/1410662?v=4
+author_name: Rutul Davé
+author_github_username: stockandawe
+---
+
+The end of calendar year is one of my favorite time of the year as I get to reflect and review the accomplishments and learnings of the past year. For the Maxwell Engineering Team, 2021 was an incredible year as we grew over 170%! Even more impressive though is how well every new contributor to our software stack exhibited [the Maxwell values](https://himaxwell.com/company/about-us) of rigor, ownership, curiosity, kindness, and being straight-up (#rocks) to make an impact towards our growth.
```

Comment:
"one of my favorite times of the year"

Reply:
I am also confused by if it is "favorite time"or "favorite times" -- thanks!

Reply:
If you say "one of" then it would be part of a set, so I think "favorite times" would make sense.